Bug: test_bgp_command Failed to run docker command

### Is it platform specific

generic

### Importance or Severity

Medium

### Description of the bug

The test is failed because it try to run `docker exec -i bgp vtysh -c "show bgp ipv4 all"` on IPv6 testbed.
Which it will return nothing. It's technically not failed to run. 

### Steps to Reproduce

Have a IPv6 topo testbed, then run:
`yanmo@sonic-mgmt-yanmo_debug:/var/src/sonic-mgmt-int/tests$ ./run_tests.sh -c bgp/test_bgp_command.py  -n vms71-t0-7060x6-3 -i ../ansible/str4,../ansible/veos -q 1 -u `



### Actual Behavior and Expected Behavior

Right now it will run IPv4 test on IPv6 testbed.
Expected :
Since the test is already pre-set to have IPv4 and IPv6 run.
It should not run IPv4 test on IPv6 testbed.

### Relevant log output

```shell
bgp/test_bgp_command.py::test_bgp_network_command[str4-7060x6-64pe-10-ipv4] FAILED [ 50%]
bgp/test_bgp_command.py::test_bgp_network_command[str4-7060x6-64pe-10-ipv6] PASSED [100%]DEBUG:tests.conftest:[log_custom_msg] item: <Function test_bgp_network_command[str4-7060x6-64pe-10-ipv6]>
INFO:root:Can not get Allure report URL. Please check logs


=================================== FAILURES ===================================
______________ test_bgp_network_command[str4-7060x6-64pe-10-ipv4] ______________

duthosts = [<MultiAsicSonicHost str4-7060x6-64pe-10>]
enum_rand_one_per_hwsku_frontend_hostname = 'str4-7060x6-64pe-10'
ip_version = 'ipv4'

    @pytest.mark.parametrize("ip_version", ["ipv4", "ipv6"])
    def test_bgp_network_command(
        duthosts, enum_rand_one_per_hwsku_frontend_hostname, ip_version
    ):
        """
        @summary: This test case is to verify the output of "show ip bgp network" command
        if it matches the output of "docker exec -i bgp vtysh -c show bgp ipv4 all" command
        """
        duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
        if ip_version == "ipv4":
            bgp_network_cmd = "show ip bgp network"
            bgp_docker_cmd = 'docker exec -i bgp vtysh -c "show bgp ipv4 all"'
        elif ip_version == "ipv6":
            bgp_network_cmd = "show ipv6 bgp network"
            bgp_docker_cmd = 'docker exec -i bgp vtysh -c "show bgp ipv6 all"'
    
        bgp_network_result = duthost.shell(bgp_network_cmd)
        bgp_network_output = bgp_network_result["stdout"]
        pytest_assert(
            bgp_network_result["rc"] == 0,
            "{} return value is not 0, output={}".format(
                bgp_network_cmd, bgp_network_output
            ),
        )
        pytest_assert(
            "*=" in bgp_network_output or "*>" in bgp_network_output,
            "Failed to run '{}' command, output={}".format(
                bgp_network_cmd, bgp_network_output
            ),
        )
    
        bgp_docker_result = duthost.shell(bgp_docker_cmd)
        bgp_docker_output = bgp_docker_result["stdout"]
        pytest_assert(
            bgp_docker_result["rc"] == 0,
            "{} return value is not 0, output:{}".format(bgp_docker_cmd, bgp_docker_output),
        )
>       pytest_assert(
            "*=" in bgp_docker_output or "*>" in bgp_docker_output,
            "Failed to run '{}' command, output={}".format(
                bgp_docker_cmd, bgp_docker_output
            ),
        )
E       Failed: Failed to run 'docker exec -i bgp vtysh -c "show bgp ipv4 all"' command, output=
```

### Output of `show version`

```shell
admin@str4-7060x6-64pe-10:~$ show ver

SONiC Software Version: SONiC.20241211.30
SONiC OS Version: 12
Distribution: Debian 12.9
Kernel: 6.1.0-29-2-amd64
Build commit: 69ca429d20
Build date: Thu Jul 10 23:33:34 UTC 2025
Built by: azureuser@cfcbcdc8c000002

Platform: x86_64-arista_7060x6_64pe
HwSKU: Arista-7060X6-64PE-C256S2
ASIC: broadcom
ASIC Count: 1
Serial Number: HBG2441052R
Model Number: DCS-7060X6-64PE
Hardware Revision: 02.00
Uptime: 01:35:16 up 2 days,  5:15,  1 user,  load average: 1.47, 1.36, 1.29
Date: Tue 15 Jul 2025 01:35:16

Docker images:
REPOSITORY                 TAG           IMAGE ID       SIZE
docker-mux                 20241211.30   e8551ed5f67f   365MB
docker-mux                 latest        e8551ed5f67f   365MB
docker-sonic-telemetry     20241211.30   466740098cc3   384MB
docker-sonic-telemetry     latest        466740098cc3   384MB
docker-macsec              latest        691e75ee9787   346MB
docker-dhcp-server         latest        bb7ef444cc11   336MB
docker-eventd              20241211.30   6593878b532b   313MB
docker-eventd              latest        6593878b532b   313MB
docker-sonic-gnmi          20241211.30   4b75127630d8   383MB
docker-sonic-gnmi          latest        4b75127630d8   383MB
docker-gbsyncd-broncos     20241211.30   bfa7872a2768   355MB
docker-gbsyncd-broncos     latest        bfa7872a2768   355MB
docker-gbsyncd-credo       20241211.30   5479034784af   328MB
docker-gbsyncd-credo       latest        5479034784af   328MB
docker-orchagent           20241211.30   724cae4c3d78   356MB
docker-orchagent           latest        724cae4c3d78   356MB
docker-fpm-frr             20241211.30   c2fb97ce94df   371MB
docker-fpm-frr             latest        c2fb97ce94df   371MB
docker-teamd               20241211.30   fe4c78629b31   343MB
docker-teamd               latest        fe4c78629b31   343MB
docker-sonic-bmp           20241211.30   be59a4b630f9   315MB
docker-sonic-bmp           latest        be59a4b630f9   315MB
docker-dhcp-relay          latest        c779f94f437b   324MB
docker-platform-monitor    20241211.30   654e77a1ffef   434MB
docker-platform-monitor    latest        654e77a1ffef   434MB
docker-snmp                20241211.30   76fe6025f4e0   358MB
docker-snmp                latest        76fe6025f4e0   358MB
docker-database            20241211.30   b0d4226894da   322MB
docker-database            latest        b0d4226894da   322MB
docker-acms                20241211.30   467ad4365d95   364MB
docker-acms                latest        467ad4365d95   364MB
docker-syncd-brcm          20241211.30   0ae39b2fae3d   798MB
docker-syncd-brcm          latest        0ae39b2fae3d   798MB
docker-router-advertiser   20241211.30   25a1f85f3891   314MB
docker-router-advertiser   latest        25a1f85f3891   314MB
docker-lldp                20241211.30   6f6558fde99e   359MB
docker-lldp                latest        6f6558fde99e   359MB
docker-sonic-restapi       20241211.30   0a2a3b4e07a6   332MB
docker-sonic-restapi       latest        0a2a3b4e07a6   332MB
k8s.gcr.io/pause           3.5           ed210e3e4a5b   683kB

admin@str4-7060x6-64pe-10:~$
```

### Attach files (if any)

_No response_

--- a/tests/bgp/test_bgp_command.py
+++ b/tests/bgp/test_bgp_command.py
@@ -32,16 +32,16 @@ def test_bgp_network_command(
     if it matches the output of "docker exec -i bgp vtysh -c show bgp ipv4 all" command
     """
     duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
-    # Determine if we are on IPv6 topology
-    ipv6_topo = (
+    # Determine if we are on IPv6 only topology
+    ipv6_only_topo = (
         "-v6-" in tbinfo["topo"]["name"]
         if tbinfo and "topo" in tbinfo and "name" in tbinfo["topo"]
         else False
     )
 
     if ip_version == "ipv4":
-        if ipv6_topo:
-            pytest.skip("Skipping IPv4 BGP network command test in IPv6 topology")
+        if ipv6_only_topo:
+            pytest.skip("Skipping IPv4 BGP network command test in IPv6 only topology")
         bgp_network_cmd = "show ip bgp network"
         bgp_docker_cmd = 'docker exec -i bgp vtysh -c "show bgp ipv4 all"'
     elif ip_version == "ipv6":

--- a/tests/bgp/test_bgp_command.py
+++ b/tests/bgp/test_bgp_command.py
@@ -25,14 +25,23 @@ def parse_routes_and_paths(output):
 
 @pytest.mark.parametrize("ip_version", ["ipv4", "ipv6"])
 def test_bgp_network_command(
-    duthosts, enum_rand_one_per_hwsku_frontend_hostname, ip_version
+    duthosts, enum_rand_one_per_hwsku_frontend_hostname, ip_version, tbinfo
 ):
     """
     @summary: This test case is to verify the output of "show ip bgp network" command
     if it matches the output of "docker exec -i bgp vtysh -c show bgp ipv4 all" command
     """
     duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
+    # Determine if we are on IPv6 topology
+    ipv6_topo = (
+        "-v6-" in tbinfo["topo"]["name"]
+        if tbinfo and "topo" in tbinfo and "name" in tbinfo["topo"]
+        else False
+    )
+
     if ip_version == "ipv4":
+        if ipv6_topo:
+            pytest.skip("Skipping IPv4 BGP network command test in IPv6 topology")
         bgp_network_cmd = "show ip bgp network"
         bgp_docker_cmd = 'docker exec -i bgp vtysh -c "show bgp ipv4 all"'
     elif ip_version == "ipv6":

